Pacemaker/pacemaker_remote_poc/README.md

```diff
@@ -0,0 +1,36 @@
+### Conceptual Overview
+
+- A regular pacemaker cluster with communication using corosync can be extended up to 32 nodes. With the pacemaker_remote service, High Availability clusters can be extended to include additional nodes beyond this limit.
+- The pacemaker_remote service can be operated as a physical node (remote node) or as a virtual node (guest node). Unlike normal cluster nodes, both remote and guest nodes are managed by the cluster as resources.
+- From the resource management point of view, they behave as regular cluster nodes.
+- Remote nodes do not need to have the full cluster stack installed, as they only run the pacemaker_remote service. The service acts as a proxy, allowing the cluster stack on the“regular”cluster nodes to connect to the service.
+
+### Terminology
+###### **Cluster Node**: A node that runs the complete cluster stack.
+A regular cluster node can perform following:
+- Run cluster resources.
+- Run all command line tools, such as crm , crm_mon.
+- Execute fencing actions.
+- Count toward cluster quorum.
+- Serve as the designated coordinator (DC).
+
+###### **Pacemaker Remote (systemd service: pacemaker_remote )**
+- A service daemon that makes it possible to use a node as a Pacemaker node without deploying the full cluster stack.
+
+###### **Remote Node: A physical machine that runs the pacemaker_remote daemon**
+- A special resource( ocf:pacemaker:remote ) needs to run on one of the cluster nodes to manage communication between the cluster node and the remote node
```

Comment:
Not everything.
1. It is itself added as a resource in a cluster. You can run some services on that node which will be managed my pacemaker then as I described above
2. You can execute command line tools crm_mon etc on a  remote node
3. Remote node can be fenced
4. It can not be served as DC and will not take part in quorum.

Reply:
I would suggest to explicitly mention the limitations for remote nodes in the document. 
Rationale is simple: since there is the only "feature" listed for remote node, it makes false impression that remote node can do absolutely the same as cluster node described above + specified one.

Reply:
Not sure if they are limitations, but yes, I agree, differences needs to be highlighted. And I specifically mentioned that. Please let me know still there is a space for improvement.
Thanks !